native/android/app/build.gradle

```diff
@@ -128,6 +129,14 @@ android {
             include "armeabi-v7a", "x86"
         }
     }
+    productFlavors {
```

Comment:
There's a whole different way to achieve this in the Android-Gradle plugin: https://developer.android.com/studio/build/configure-apk-splits#configure-abi-split.  Explain to me what the differences between this approach and that approach are?